### PR TITLE
Prepare 2.14.0 release

### PR DESCRIPTION
**Pull request by JillieBeanSim**
## Proposed changes

Prepare 2.14.0 release

## Release Notes

Milestone: 2.14.0

Changelog: n/a

## Types of changes

What types of changes does your code introduce to Zowe Explorer?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Updates to Documentation or Tests (if none of the other choices apply)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This checklist will be used as reference for both the contributor and the reviewer_

- [x] I have read the [CONTRIBUTOR GUIDANCE](https://github.com/zowe/vscode-extension-for-zowe/wiki/Best-Practices:-Contributor-Guidance) wiki
- [x] PR title follows [Conventional Commits Guidelines](https://www.conventionalcommits.org/en/v1.0.0-beta.2/)
- [x] PR Description is included
- [ ] gif or screenshot is included if visual changes are made
- [x] `yarn workspace vscode-extension-for-zowe vscode:prepublish` has been executed
- [x] All checks have passed (DCO, Jenkins and Code Coverage)
- [ ] I have added unit test and it is passing
- [ ] I have added integration test and it is passing
- [ ] There is coverage for the code that I have added
- [ ] I have tested it manually and there are no regressions found
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any PR dependencies have been merged and published (if appropriate)

### CODE REVIEW

**Comment by JillieBeanSim** [comment]
@zFernand0 was there any updates needed in docs or the README for these changes that are highlighted?
Screenshot 2024-01-23 at 8 20 45 AM

File: packages/zowe-explorer/README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 > ## v3 Pre-release is now available in our [Github Releases](https://github.com/zowe/vscode-extension-for-zowe/releases) with the removal of v1 profile support. Keep an eye on [changes affecting users and extenders](https://github.com/zowe/vscode-extension-for-zowe/wiki/v3-Changes-for-Users-and-Extenders) for the full list of changes
 >
-> ## With the removal of keytar shim from VS Code we will be replacing `node-keytar` with `@zowe/secrets-for-zowe-sdk` in the v2.10.0 release to avoid user interruptions. Extenders, please check out the usage [documentation](https://github.com/zowe/vscode-extension-for-zowe/wiki/Secure-Credentials-for-Extenders)
+> ## With the removal of keytar shim from VS Code we have replaced `node-keytar` with `@zowe/secrets-for-zowe-sdk` in the v2.10.0 release to avoid user interruptions. Extenders, please check out the usage [documentation](https://github.com/zowe/vscode-extension-for-zowe/wiki/Secure-Credentials-for-Extenders)
```
Comment (adam-wolfe) [inline packages/zowe-explorer/README.md:10]: Small suggestions:
```
"With the removal of the keytar shim from VS Code, we have..."
                     ^^^                         ^            
```

**Review by adam-wolfe — APPROVED** [review]
LGTM

**Review by traeok — APPROVED** [review]

**Review by rudyflores — APPROVED** [review]
LGTM, thanks for creating the release PR @JillieBeanSim !

**Review by t1m0thyj — APPROVED** [review]

**Pull request merged**